xt_cvdata/transforms/transforms.py

```diff
@@ -12,7 +12,8 @@
 
 class XTCompose:
     """ Custom Composer of transforms used for applying transforms to the img, mask, or both.
-    Can use both Albuentations transforms, or any transform on PIL images
+    Can use both Albuentations transforms, or any transform on PIL images.
+    Note: Images must be RGB and Mask must be A
```

Comment:
What do you mean by Mask must be A?

Reply:
Maybe it should be "Mask must be a single-channel image" or something?

Reply:
A meant alpha but agreed its unclear. Changed and merged.